tests/Oryx.Tests.Common/DockerVolume.cs

```diff
@@ -42,30 +41,35 @@ private DockerVolume(string originalHostDir, string mountedHostDir, string conta
         /// </summary>
         public string ContainerDir { get; }
 
-        public static DockerVolume Create([NotNull] string hostPath, [NotNull] string containerPath)
+        /// <summary>
+        /// Allows a container to use host's docker engine.
+        /// </summary>
+        /// <returns></returns>
+        public static DockerVolume CreateDockerSocket()
```

Comment:
@dorfire Probably I was not clear but I actually meant this when I mentioned in your earlier PR.

Reply:
this name could be a little confusing - after all, no "DockerSockets" are being created here

Reply:
Any suggestions? I initially wanted to say something like `MountDockerSocket` but that felt incorrect as you are not really mounting it at the point you might be calling this method.

Reply:
Maybe just add a static member `DockerVolume.DockerDaemonSocket`?

Reply:
Sounds good 👍